src/test/javascript/spec/service/orion/orion-assessment.service.spec.ts

```diff
@@ -114,19 +114,19 @@ describe('OrionAssessmentService', () => {
      * Helper to test the conversion with the fileReader
      * @param mockReader mock reader to test
      */
-    function testConversion(mockReader: FileReader) {
+    function testConversion(mockReader: any) {
         const isCloningSpy = jest.spyOn(orionConnectorService, 'isCloning');
-        const exportSubmissionStub = jest.spyOn(programmingAssessmentExportService, 'exportReposByParticipations');
-        const lockAndGetStub = jest.spyOn(programmingSubmissionService, 'lockAndGetProgrammingSubmissionParticipation');
-        const readerStub = jest.spyOn(window, 'FileReader');
 
         // first it loads the submission
+        const lockAndGetStub = jest.spyOn(programmingSubmissionService, 'lockAndGetProgrammingSubmissionParticipation');
         lockAndGetStub.mockReturnValue(of(programmingSubmission));
         // then the exported file
+        const exportSubmissionStub = jest.spyOn(programmingAssessmentExportService, 'exportReposByParticipations');
         const response = new HttpResponse({ body: new Blob(['Stuff', 'in blob']), status: 200 });
         exportSubmissionStub.mockReturnValue(of(response));
 
-        readerStub.mockReturnValue(mockReader);
+
```

Comment:
Please fix this error. After that, it should be fine

Reply:
Fixed. Sorry, did not notice it